Task03-OnePieceWebsite/style/layouts/_content.scss

```diff
@@ -0,0 +1,119 @@
+/**
+ * All Rights Reserved
+ * This software is property information of AKUREY S.A.
+ * Contact us at contact@akurey.com
+ * @summary short description for the file
+ */
+@import '../components/searchbar';
+@import '../sections/sipnosis';
+
+.content{
```

Comment:
remember leaving a space before '{', same for all cases in following files

Reply:
On the last PR review you mention the spaces after the brackets and i thought you meant to remove them so i remove all of them for that, im adding them right now

Reply:
Sorry! maybe I confused after with before haha, but yeah add a space before the bracket
This is tedious I know, on future projects some tools will take care of this for you! 🥳

Reply:
Haha ok ok no worries, we are cleared now